crypto/Pos/Crypto/Configuration.hs

```diff
@@ -24,13 +17,5 @@ newtype ProtocolMagic = ProtocolMagic
 deriving instance ToJSON ProtocolMagic
 deriving instance FromJSON ProtocolMagic
 
-type HasProtocolMagic = Given ProtocolMagic
-
--- | Protocol magic constant. Is put to block serialized version to
--- distinguish different clusters. It's an example, possible usages
--- are wider.
-protocolMagic :: HasProtocolMagic => ProtocolMagic
-protocolMagic = given
-
-withProtocolMagic :: ProtocolMagic -> (HasProtocolMagic => r) -> r
-withProtocolMagic = give
+dummyProtocolMagic :: ProtocolMagic
+dummyProtocolMagic = ProtocolMagic 55550001
```

Comment:
Should comment this. Is it used at all outside of testing?

Reply:
Currently this is only used in Arbitrary instances. Maybe it should be moved to the `crypto-test` package so that we'll see an explicit dependency on `crypto-test` if it gets used in the libraries. This is the same as the value defined in `configuration.yaml` - what are the plans for that in the long run?

Reply:
If its only used for Arbitrary instances it should be in the `-test` package.

Reply:
Moved to `Test.Pos.Crypto.Dummy`!